_projects/2017/glue/notebook.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Glue notebook viewers
+desc: Develop Jupyter notebook viewers for glue
```

Comment:
a general point, I think the "desc" field should be a little longer here, just looking at the way they render on the website, this is your sales pitch, it's all people see without clicking on the idea.